pages/index.tsx

```diff
@@ -13,6 +21,48 @@ export default function Home() {
       <section className={styles.container}>
         <main className={styles.MainArea}>
           <HeroBanner imagePath='logo.svg' altText='TCSCA Logo' />
+          {/* NOTE: we may want to put stuff in seperate sections rather than back to back */}
+          {homeSections.map((section, index) => {
+            const sectionStyle =
+              index % 2 === 0
+                ? SectionHeaderStyle.BLACK_ON_GREEN
+                : SectionHeaderStyle.GREEN_ON_BLACK;
+            const location =
+              index % 2 === 0
+                ? SectionLocation.LEFT_ALIGNED
+                : SectionLocation.RIGHT_ALIGNED;
+            const textSectionStyle =
+              index % 2 === 0 ? TextSectionStyle.BLACK : TextSectionStyle.GREEN;
+            return (
+              <>
+                <SectionHeader
```

Comment:
Maybe we make this into a a `Section` Component which handles matching the seciton to the child, it could also maybe take the `Section` from the config and handle mapping the config to a section.

Reply:
Fixed